### PR TITLE
Adding a React sample app showing how to use Wootric

### PR DESCRIPTION
**Pull request by gmonfort**
Adds a React example showing how to use Wootric

### CODE REVIEW

**Comment by hachibu** [comment]
Can you squash down to one commit.

File: react/src/config/defaults.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const WOOTRIC_ACCOUNT_TOKEN = 'NPS-9fbff420';
```
Comment (hachibu) [inline react/src/config/defaults.js]: Replace with fake token like NPS-XXXXXXXX

Comment (gmonfort) [inline react/src/config/defaults.js]: Ah yes thanks!

File: react/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Installation
+
+* `git clone <repository-url>` this repository
+* `cd react`
+* `npm install`
+
+## Configuring Wootric
+* Open `src/config/defaults.js`.
+* Update `WOOTRIC_ACCOUNT_TOKEN` with your account token.
```
Comment (hachibu) [inline react/README.md]: Nice 👍

File: react/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Prerequisites
+
+You will need the following things properly installed on your computer.
+
+* [Git](https://git-scm.com/)
+* [Node.js](https://nodejs.org/) (with NPM)
+
+## Installation
+
+* `git clone <repository-url>` this repository
```
Comment (hachibu) [inline react/README.md]: You should paste in the url here instead of saying <repository-url>

File: react/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+## Configuring Wootric
+* Open `src/config/defaults.js`.
+* Update `WOOTRIC_ACCOUNT_TOKEN` with your account token.
+
+## Running / Development
+
+* `npm start`
+* Visit your app at [http://localhost:3000](http://localhost:3000).
+
+## Available Scripts
```
Comment (hachibu) [inline react/README.md]: I think you can remove the text from this line down to keep things simple.

**Comment by hachibu** [comment]
Fix the typo in the commit message. React instead of Ract.

**Pull request merged**